tests/fixtures/frontend.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { UserProfile, UserProvider } from '../../src';
+import { UserProvider, UserProviderProps, UserProfile } from '../../src/frontend';
```

Comment:
Could you export `UserProviderProps` from:

https://github.com/auth0/nextjs-auth0/blob/beta/src/index.ts#L95
and
https://github.com/auth0/nextjs-auth0/blob/beta/src/index.browser.ts#L5-L12

Then the developer will be able to do `import { UserProviderProps } from '@auth0/nextjs-auth0';`

And you can leave this import as `'../../src'`

Reply:
Changed in 6ceda646787bd2c0676ecc0a288522b2645e49de.